challenge.md

```diff
@@ -6,37 +6,37 @@ Write your answers inside this file, where it's indicated by the comments.
 
 1. We just forked a repo on GitHub.com and want to start working on it locally. What command do we use to do that?
 
-<!-- Answer Starts Here -->
+git clone
 
 <!-- Answer Ends Here -->
 
 2. OK, we just wrote some code. What command could we use to see a list of all the changes that have been made since the last commit?
 
-<!-- Answer Starts Here -->
+git diff
```

Comment:
Yep! We'll probably also want to use `git status` to show which files have been created, deleted, or moved.